### PR TITLE
editorial: document mandatory things needed for cr

### PR DESCRIPTION
**Pull request by marcoscaceres**
* Provides a link to a label on GitHub's issue tracker for public documentation of all substantive changes since previous publication.
* Provides a link to all commits as public documentation of changes that are not substantive.
* Documents that the dependencies during the development of the specification have not changed.
* Document how adequate implementation experience will be demonstrated (shipping implementations + test suite)
* Specifies 30 September 2017 as the deadline for CR comments.

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/subs_changes.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/bf19415...246e181.html)

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Chairs, can you please take over this PR from here? It addresses everything that needs to be document for the purpose of CR (see #520). 

The rest can be demonstrated on the day.  

Have we done the formal CFC to move the spec to CR?

**Comment by adrianhopebailie** [comment]
> Chairs, can you please take over this PR from here? It addresses everything that needs to be document for the purpose of CR (see #520).

Thanks. Will do

> Have we done the formal CFC to move the spec to CR?

No, we are waiting for the go-ahead from the editors. This is on the agenda for tomorrow's call.

File: index.html [hunk 1]
```diff
@@ -2864,7 +2875,31 @@
...
+        creating a <a href=
+        "https://github.com/w3c/web-platform-tests/tree/master/payment-request">
+        test suite</a> and having at least two independent implementations pass
+        each mandatory test (i.e., each test the corresponds to a MUST
+        requirements of the specification). The working group hopes to
+        demonstrate, in the form of an implementation report, interoperability
+        from two or more vendors on both mobile and desktop web browsers.
+      </p>
+      <p>
+        There have been no change in dependencies on other workings groups
```
Comment (ianbjacobs) [inline index.html]: "There have been no changes"

or

"There has been no change"

**Review by ianbjacobs — COMMENTED** [review]
I have suggested a minor editorial fix. Otherwise ready to go.

File: index.html [hunk 2]
```diff
@@ -86,9 +98,8 @@
         are strongly encouraged.
       </p>
       <p>
-        This specification was derived from a report published previously by
-        the <a href="https://www.w3.org/community/wicg/">Web Platform Incubator
-        Community Group</a>.
+        The deadline for comments for Candidate Recommendation is 31 December
```
Comment (marcoscaceres) [inline index.html]: Spoke to @adrianba, we agreed to 6 weeks + 3 months from publication.

**Review by zkoch — APPROVED** [review]

**Pull request merged**